networkx/algorithms/shortest_paths/unweighted.py

```diff
@@ -30,30 +30,27 @@ def single_source_shortest_path_length(G, source, cutoff=None):
        Starting node for path
 
     cutoff : integer, optional
-        Depth to stop the search. Only paths of length <= cutoff are returned.
+        Depth to stop the search. Only paths of length <= `cutoff` are returned.
 
     Returns
     -------
     lengths : dict
-        Dict keyed by node to shortest path length to source.
+        Dict keyed by node to shortest path length to `source`.
 
     Examples
     --------
     >>> G = nx.path_graph(5)
-    >>> length = nx.single_source_shortest_path_length(G, 0)
-    >>> length[4]
-    4
-    >>> for node in length:
-    ...     print(f"{node}: {length[node]}")
-    0: 0
-    1: 1
-    2: 2
-    3: 3
-    4: 4
+    >>> nx.single_source_shortest_path_length(G, 0)
+    {0: 0, 1: 1, 2: 2, 3: 3, 4: 4}
 
     See Also
     --------
-    shortest_path_length
+    :func:`~networkx.algorithms.shortest_paths.generic.shortest_path_length` :
+       Shortest path length with specifiable source, target, and weight.
+    :func:`~networkx.algorithms.shortest_paths.weighted.single_source_dijkstra_path_length` :
+       Shortest weighted path length from source with Dijkstra algorithm.
+    :func:`~networkx.algorithms.shortest_paths.weighted.single_source_bellman_ford_path_length` :
+       Shortest weighted path length from source with Bellman-Ford algorithm.
```

Comment:
Do we need single backticks in these see-also descriptions too? And should the indent be 4 chars?

And would it be better to describe when they should use djikstra/bellman-ford rather than just give the name? (And maybe it is best to use the single line description from the function itself for consistency across "See-also" sections. Sphinx does that for automodule... but I guess not for See Also)
```suggestion
    :func:`~networkx.algorithms.shortest_paths.generic.shortest_path_length` :
        Shortest path length with specifiable source, target, and weight.
    :func:`~networkx.algorithms.shortest_paths.weighted.single_source_dijkstra_path_length` :
        Shortest weighted path length from `source` for positive weights.
    :func:`~networkx.algorithms.shortest_paths.weighted.single_source_bellman_ford_path_length` :
        Shortest weighted path length from `source` for possibly negative weights.
```

To be clear, these nit-picks aren't needed for this PR. But I bring them up because we're doing a deep-dive in prep for rewriting all these doc_strings.

Reply:
> Do we need single backticks in these see-also descriptions too?

Yep - that was an oversight on my part - good catch!

> And should the indent be 4 chars?

Aside from style concerns it doesn't matter. I'm used to 3-spaces for `rst`, which I think is the most common level of indentation (and which AFAIK is a convention stemming  from the fact that directives start with `.. `, and folks like to align text with directive names). However, 4 is also fine - I don't believe we have/enforce any style rules on this front, but I'd be in favor of doing so (automatically, that is!)

> And would it be better to describe when they should use djikstra/bellman-ford rather than just give the name? 

Absolutely - I just wasn't confident about when one might be preferred over :upside_down_face: